### DISCUSSION TITLE
Postal Database Config

### DISCUSSION THREAD

**Discussion by jASSSSSSON** [postalserver/postal#2421] [Help with using Postal]
Hello everyone, this is my first time using Postal. When I executed the installation instructions on the official website, I hoped to be able to use an external database for operation. However, its port was not 3306, so I tried to directly use the port in the host, but it became xxx. xxx. xxx. xxx: yyyy: 3306. I also tried to directly add the port attribute, but it did not take effect. What should I do? Or where can I view detailed instructions? Thank you.

**Comment by willpower232** [comment]
Hopefully you've seen that the configuration file provides port as a separate setting but don't forget there are two places for it.

https://github.com/postalserver/postal/blob/5222263484f2a5889653859db6aeb7438f116357/config/postal.defaults.yml#L27-L29

https://github.com/postalserver/postal/blob/5222263484f2a5889653859db6aeb7438f116357/config/postal.defaults.yml#L44-L46
